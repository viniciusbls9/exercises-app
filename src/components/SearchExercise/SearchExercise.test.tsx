import { render, screen } from '@testing-library/react';
import ExerciseSearch from './';

describe('ExerciseSearch', () => {
  it('renders the text and input field', () => {
    const { container } = render(<ExerciseSearch />);
    const textElement = screen.getByText('Find your exercises');
    const inputElement = screen.getByPlaceholderText('Enter your exercise');
    expect(textElement).toBeInTheDocument();
    expect(inputElement).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  })
});