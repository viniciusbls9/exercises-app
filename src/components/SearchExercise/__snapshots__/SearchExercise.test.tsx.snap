// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ExerciseSearch > renders the text and input field 1`] = `
<div>
  <div
    class="sc-aXZVg bouHrP"
  >
    <div
      class="sc-gEvEer iHaQaB"
    >
      <h2
        class="sc-eqUAAy kVCmHi"
      >
        Find your exercises
      </h2>
      <div
        class="sc-fqkvVR dKFVrM"
      >
        <input
          class="sc-dcJsrY cafawp"
          placeholder="Enter your exercise"
          type="text"
        />
        <button
          class="sc-iGgWBj hCngai"
        >
          Search
        </button>
      </div>
    </div>
  </div>
</div>
`;
